yargs/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.editorconfig should be the root.
Please, remove this line.

ref: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/14283#discussion_r98323046